lib/main_common.pm

```diff
@@ -309,6 +309,8 @@ sub replace_opensuse_repos_tests {
 sub is_updates_tests {
     my $flavor = get_var('FLAVOR');
     return 0 unless $flavor;
+    # Add sle16 increments tests
+    return $flavor =~ /Online-Increments/ if is_sle('>=16');
```

Comment:
Online Increments exist only for sle16,  return can be:
`return $flavor =~ /-Updates|-Incidents|Online-Increments/;`